.github/workflows/deployminder.yml

```diff
@@ -0,0 +1,25 @@
+name: Remind to Pull Latest Changes
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  remind:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment on PR
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: '@JacobCoffee Don\'t forget to pull the latest changes on `salt.nyc1.psf.io`!'
```

Comment:
Heh, if you want the mention! I don't think it needs to be this targeted cause on your merges you would get an email notification of the comment regardless, right?